src/Histogram/Histogram.js

```diff
@@ -480,17 +487,17 @@ export class Histogram extends PureComponent {
      * @returns {React.Element}
      */
     _renderDensityChart() {
-        const { densityChartDimensions, brushDomain } = this.state;
         const { frameStep, frameDelay, xAccessor, spaceBetweenCharts, brushDensityChartColor,
             brushDensityChartFadedColor, renderPlayButton, data } = this.props;
 
         return (
             <DensityChart
-                width={densityChartDimensions.width}
-                height={densityChartDimensions.height}
+                width={this.state.densityChartDimensions.width}
+                height={this.state.densityChartDimensions.height}
                 padding={PADDING}
-                brushDomainMax={dateToTimestamp(brushDomain.max)}
-                brushDomainMin={dateToTimestamp(brushDomain.min)}
+                brushDomainMin={this.state.brushDomain.min}
+                brushDomainMax={this.state.brushDomain.max}
+                timeDomainMax={this.state.timeDomain.max}
```

Comment:
What is the difference between timeDomain and brushDomain?

Reply:
Ok, I need to add more information to this! 

So, the `timeDomain` is related to the full range of the data  given to the histogram component.

Please see this example
```json
[
    { "timestamp":  1, "total": 1  },
    { "timestamp":  2, "total": 1  },
    { "timestamp":  3, "total": 1  },
    { "timestamp":  4, "total": 1  },
    { "timestamp":  5, "total": 1  },
    { "timestamp":  6, "total": 1  }
]
```

The `timeRange` has  the domain between `[ 1, 6 ]`  and the `brushDomain` could be anything between this values  .

Reply:
I think the name `timeDomain` is not very good, the `brushDomain` is also a time domain. Why not `overallDomain` or `overallDataDomain` or something that indicates that this related to all the data?